web_src/css/repo/header.css

```diff
@@ -29,11 +29,13 @@
 }
 
 .repo-buttons {
-  align-items: center;
-  display: flex;
   flex-flow: row wrap;
   word-break: keep-all;
-  gap: 0.25em;
+}
+
+.repo-buttons button {
+  /* ensure that text-btn and svg-text-btn have consistent height. */
+  line-height: 16px !important;
```

Comment:
Why it is needed? I don't see difference.

Reply:
OK, I can see that it is for the "accept/reject transfer" buttons

Reply:
This patch should also be refactored together later.

![image](https://github.com/user-attachments/assets/37209960-6464-4170-85bb-61aa21c9834a)